mlir/include/Quantum/Transforms/Passes.td

```diff
@@ -117,6 +117,13 @@ def MergeRotationsPass : Pass<"merge-rotations"> {
     let options = QuantumCircuitTransformationPass.options;
 }
 
+def IonsDecompositionPass : Pass<"ions-decomposition"> {
+    let summary = "Decompose the gates to the set {RX, RY, MS}";
+
+    let constructor = "catalyst::createIonsDecompositionPass()";
+    let options = QuantumCircuitTransformationPass.options;
```

Comment:
This line inherits the `func-name` option into the pass. See my other comment.

Reply:
Good catch, I will remove it from this PR.